core/templates/components/question-directives/questions-list/questions-list.directive.html

```diff
@@ -29,13 +64,61 @@ <h3>Question List</h3>
       </div>
     </div>
   </md-card>
-  <div class="create-question-container">
+  <md-card class="question-data-container question-editor-open-container" ng-if="$ctrl.editorIsOpen">
+    <div class="save-question-container">
+      <div>
+        <div class="list-header">
+          <h3>New Question</h3>
+        </div>
+        <div class="checklist-item">
+          <div class="checklist-item-number" ng-class="{'checklist-item-number-active': linkedSkillsWithDifficulty.length}"><span>1</span></div>
```

Comment:
Should it be  $ctrl.linkedSkillsWithDifficulty.length ?

Reply:
Yes fixed.